src/features/burn/BurnScreen.tsx

```diff
@@ -93,17 +92,14 @@ const BurnScreen = () => {
   const addressBookRef = useRef<AddressBookRef>(null)
   const {
     floatToBalance,
-    dcToNetworkTokens,
     networkTokensToDc,
     hntBalance,
     solBalance,
     dcBalance,
   } = useBalance()
-  const [fee, setFee] = useState<Balance<DataCredits>>()
   const { makeBurnTxn } = useTransactions()
   const { showOKAlert } = useAlert()
   const hntKeyboardRef = useRef<HNTKeyboardRef>(null)
-  const { l1Network } = useAppStorage()
```

Comment:
Should we remove ledger or show an error for now?

Reply:
Hmm let me look into this

Reply:
Hmm there is so much ledger stuff. I'm going to leave that for another ticket